contracts/beacon/Voucher.sol

```diff
@@ -295,6 +290,23 @@ contract Voucher is OwnableUpgradeable, IVoucher {
         return $._sponsoredAmounts[dealId];
     }
 
+    /**
+     * Retrieve the address of the voucher hub associated with the voucher.
+     * @return voucherHubAddress The address of the voucher hub.
+     */
+    function getVoucherHub() public view returns (address) {
+        VoucherStorage storage $ = _getVoucherStorage();
+        return $._voucherHub;
+    }
+
+    /**
+     * @dev Returns the address of the current owner.
+     */
+    function owner() public view returns (address) {
```

Comment:
Minor, it might also make sense to move up ower() a bit for better consistency with fields